test/native/setup.js

```diff
@@ -122,8 +122,9 @@ jest.mock(
 jest.mock( 'react-native-hr', () => () => 'Hr' );
 
 jest.mock( 'react-native-svg', () => {
+	const { forwardRef } = require( 'react' );
 	return {
-		Svg: () => 'Svg',
+		Svg: forwardRef( mockComponent( 'Svg' ) ),
```

Comment:
Social Icon block uses the animated version of the SVG component, which requires the `ref` prop. Hence we need mock the component to support that prop:

- https://github.com/WordPress/gutenberg/blob/cbd4fbbd60b634a4f38f7c17ea0fa72fa3deb3b4/packages/block-library/src/social-link/edit.native.js#L201-L209
- https://github.com/WordPress/gutenberg/blob/cbd4fbbd60b634a4f38f7c17ea0fa72fa3deb3b4/packages/primitives/src/svg/index.native.js#L30-L32